### PR TITLE
elf: don't assume all maps are perf maps

### PR DESCRIPTION
**Pull request by iaguis**
cc @alban

### CODE REVIEW

**Comment by alban** [comment]
This issue was discovered via https://github.com/weaveworks/scope/pull/2135#issuecomment-279901474
Symptoms:
> error initializing tcptracer_status map: unable to update element: argument list too long

**Comment by alban** [comment]
CI fails.

@schu any idea what's happening? I don't see obvious error.

**Comment by schu** [comment]
>  any idea what's happening? I don't see obvious error.

No. Green now, after triggering a new build.

File: elf/elf.go [hunk 1]
```diff
@@ -478,9 +478,13 @@ func (b *Module) Load() error {
 		}
 	}
 
-	for name, _ := range b.maps {
+	for name, m := range b.maps {
```
Comment (alban) [inline elf/elf.go]: could we move this block in a separate function? So the name of the function would make it clear this is about initializing the perf maps, and not just generic elf parsing

Comment (iaguis) [inline elf/elf.go]: Done

**Pull request merged**